samples/Tests/TextBlock.MultipleFonts.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.1",
```

Comment:
>1.1 [](start = 13, length = 3)

Please update the version of this card to 1.2. Also, i've created a 1.2 directory we can move this into. For naming, i think we'd want to call it TextBlock.FontStyle.json. I believe that causes the site to magically pick it up as a sample for the fontstyle property documentation.